styles/index.css

```diff
@@ -5,7 +5,9 @@
 @tailwind utilities;
 @tailwind utilities;
 
-
+.active{
+    color: #0F27BD;
+}
```

Comment:
I don't think this is necessary here, you can add it to the file you made changes to, so this won't affect other people's work with the same class.

Reply:
Alright. I will do just that